### PR TITLE
Update all.json

### PR DESCRIPTION
**Pull request by dubstard**
<html>
<body>

Site | URLScan link
-- | --
apps-syncs.org|https://user-images.githubusercontent.com/49607867/170858165-2e298be9-f13f-41ac-829e-e4946b5efe77.png
kusama.lida.finance | https://urlscan.io/result/28be361d-9190-4380-8d41-4ba52c4f1a7e/
solutioncoiinmsmartteconnet.live|https://user-images.githubusercontent.com/49607867/170857775-9939b93e-6887-43fc-a3d0-9ae3dbbb40d0.png
encryptodapps.pages.dev|https://urlscan.io/result/ac39d4dc-7f5d-4244-9afc-f47fba95ac0d/dom/
tokenimtu.top|https://user-images.githubusercontent.com/49607867/170857901-0dde732e-0ebc-429a-bc2e-48cfcbb732bd.png

</body>
</html>

```
		"apps-syncs.org",
		"app-wallets.ml",
		"chain-node.org",
		"dappcryptostore.com",
		"encryptodapps.pages.dev",
		"kusama.lida.finance",
		"lida.finance",
		"mainnet-migrate.online",
		"metamask-recovery.info",
		"metamask-recovery.net",
		"metamask-update-v2.com",
		"metamask.io-downloadproductwallettokens.io-vebsite.xyz",
		"metamask.io-downloadproductwallettokenssk.jo-website.xyz",
		"metamaskondemand.com",
		"metamaskweb.org",
		"metamusk.app-wallets.ml",
		"opensea-nft-offers.com",
		"opensea-verificationprocess.com",
		"pancakeswap-dapps.com",
		"secure-opensea-x03.com",
		"solutioncoiinmsmartteconnet.live",
		"stepend.art",
		"support-opansea.io",
		"tokenimtu.top",
		"updatingmyopensea.com",
		"wallet.metamaskweb.org",
		"x0384-open-sea-recover.com",

### CODE REVIEW

**Review by jacogr — APPROVED** [review]

**Comment by dubstard** [comment]
Thanks

**Comment by jacogr** [comment]
… gawd, your PRs take a shitload of time to review. Half the entries :)

**Comment by dubstard** [comment]
Sorry, me being sloppy and overzealous resulted in a lot of false positives for May, which were not targeting dot explicitly, but i marked them as such, due to my error (manual entering a param as part of the URL as part of fuzzing)

This generated a lot of double work for the curator and council as well, had to mark as non eligible for the child bounty.

I shall be be much more strict from now on and limit the PRs to relevant stuff only, as in not to cause unnecessary burden.

**Comment by jacogr** [comment]
I'm not complaining ... love your work, but smaller PRs are easier to review. (For me at least - since I check each URL)

**Comment by dubstard** [comment]
Noted with thanks! 🙏

**Comment by dubstard** [comment]
Ah, I am afraid I spoke too soon about being careful in the future.
5,538 sites from ~ 10K - oh well it was a fun ride!

Thanks and sorry to lose trust.

**Comment by jacogr** [comment]
My only complaint is on the review, not the volume of the items that actually go im. So what I'm actually asking for greater volume of PRs with smaller number of entries per PR. (Just makes the turn-around, for me, slightly faster - since I do check them all... just as much of a horrid job checking than it is adding them in the first place.)

It is easier to review 15 entries than it is to review 50 - not due to me being lazy, just rather due to me being old and losing track along the way :) My only concern with slightly more PRs is that there may sneak in conflicts, we have seen that before.

TL;DR Not bitching about 10k at all, just want to allow myself to have a slightly faster turn-around on per-PR reviews.
